src/components/BooksList.js

```diff
@@ -0,0 +1,26 @@
+import PropTypes from 'prop-types';
+
+const Bookslist = ({ books, handleDelete }) => (
+  <div className="booksList">
+    {books.map((book) => (
+      <div className="prevBooks" key={book.id}>
+        {book.title}
+        {book.author}
+        {book.category}
+        <button type="button" onClick={() => handleDelete(book.id)}>
```

Comment:
- Please render a form to add a new book to the list of books. It shouldn't do anything at this stage.